group_project_v2/utils.py

```diff
@@ -269,3 +269,10 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.field._values = self._old_values_value
         return False
+
+
+def add_resource(block, resource_type, path, fragment):
```

Comment:
The block parameter is unused, so I suggest removing it.

I'm not sure I like dispatching based on a string parameter.  How about implementing two one-line functions instead?

    def add_javascript_resource(fragment, path):
        fragment.add_javascript(loader.load_unicode(path))

    def add_css_resource(fragment, path):
        fragment.add_css(loader.load_unicode(path))

Another option would be to dispatch based on the file extension, but I don't feel it would be worth it.